spec/acceptance/suites/default/00_default_spec.rb

```diff
@@ -15,12 +15,19 @@
     EOS
   }
 
-  context 'default parameters (no pki)' do
-
+  context 'setup' do
     it 'should install epel' do
       install_package(server, 'epel-release')
     end
 
+    it 'should change ownership of PKI files' do
+      # vsftpd won't start without this
+      on(server, 'chown -R root:ftp /etc/pki/simp-testing/pki')
```

Comment:
Would it more more appropriate to use `pki::copy` in the test, similar to how we use it on a full SIMP system?

```
simp_options::pki: true
simp_options::pki::source: '/etc/pki/simp-testing/pki'
```